src/Uno.UWP/Storage/CachedFileManager.wasm.cs

```diff
@@ -0,0 +1,48 @@
+using System;
+using System.IO;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+using Uno.Foundation;
+using Windows.Foundation;
+using Windows.Foundation.Metadata;
+using Windows.Storage.Provider;
+
+namespace Windows.Storage
+{
+	public partial class CachedFileManager
+	{
+		public static void DeferUpdates(IStorageFile file)
+		{
+			// The method does nothing in wasm since we don't really acces the filesystem.
+		}
+
+
+		public static IAsyncOperation<FileUpdateStatus> CompleteUpdatesAsync(IStorageFile file) => DownloadFile(file).AsAsyncOperation();
+
+		private async static Task<FileUpdateStatus> DownloadFile(IStorageFile file)
+		{
+			var stream = await file.OpenStreamForReadAsync();
+			var data = new byte[(int)stream.Length];
```

Comment:
Codacy found an issue: [Remove this useless assignment to local variable 'data'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5702699)